replications/internal/queue_management_test.go

```diff
@@ -33,83 +29,39 @@ func TestCreateNewQueueDirExists(t *testing.T) {
 }
 
 func TestEnqueueScanLog(t *testing.T) {
```

Comment:
I would drop the `Log` from these tests names since they aren't using the log output anymore.